### PR TITLE
Feature/wrapper last seen

### PR DESCRIPTION
**Pull request by mrSelieznov**
#226 Task

added wrapper last seen component

//add after service, context, component set. bar

### CODE REVIEW

**Review by mrSelieznov — COMMENTED** [review]
So mach files due to prettier

File: src/components/Home.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useState } from "react";
```
Comment (oleksandrstarov) [inline src/components/Home.tsx:1]: use single quotes

File: src/components/RenderResults.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
-import { useState } from 'react';
-import { MovieCard } from '../models/index';
-import CardInfo from './CardInfo';
-import SmallInfoCard from './SmallInfoCard';
-import useDeviceDetect from '../hooks/useDeviceDetect';
+import { useState } from "react";
```
Comment (oleksandrstarov) [inline src/components/RenderResults.tsx:1]: Use single quotes

File: src/components/RenderResults.tsx [hunk 3]
```diff
@@ -25,12 +25,12 @@ const RenderResults = ({ list }: RenderResultsArrList) => {
...
         </div>
       </div>
-    )
+    );
   }
 
   return (
     <div className="delivery">
-      <div className="view-toggler">
+      <div className="view-toggler ">
```
Comment (oleksandrstarov) [inline src/components/RenderResults.tsx:33]: remove empty space

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]
Nice refactoring, but please check your own code before creating PR

File: src/components/WrapperLastSeen.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  id: number;
+}
+
+const WrapperLasSeen = () => {
+  const { historyContext }: SettingsBarContextType = useContext(SettingsBarContext);
+
+  const { theme }: ThemeContextType = useContext(ThemeContext);
+  const [history, setHistory] = useState<MoviesType[]>();
+
+
```
Comment (oleksandrstarov) [inline src/components/WrapperLastSeen.tsx:32]: Remove line

File: src/components/WrapperLastSeen.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import { ThemeContext, ThemeContextType } from './ThemeContext';
+
+import Title from './Title';
+import Container from './Container';
+import Slider from './Slider';
+import SmallInfoCard from './SmallInfoCard';
+
+import api from '../service/api';
+
+type MovieId = {
```
Comment (oleksandrstarov) [inline src/components/WrapperLastSeen.tsx:13]: I remember I left comment regarding such type - we have one same  already and can reuse it

**Review by oleksandrstarov — APPROVED** [review]
Fix comments

**Pull request merged**